src/main/java/uk/ac/ebi/ampt2d/registry/entities/Property.java

```diff
@@ -0,0 +1,107 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.registry.entities;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import org.hibernate.validator.constraints.NotBlank;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.time.ZonedDateTime;
+
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+public class Property {
+
+    public enum Type {
+
+        BOOLEAN,
+
+        INTEGER,
+
+        FLOAT,
+
+        DOUBLE,
+
+        STRING
+
+    }
+
+    public enum Meaning {
+
+        NONE,
+
+        CALL_RATE,
+
+        P_VALUE,
+
+        MAF,
+
+        MAC
+
+    }
```

Comment:
Should the meaning be enums ?

Reply:
IMHO for the moment they should be enums and they should dissappear in the future. Other oppinions?

Reply:
I assume we are maintaining it for now so the portal doesn't completely break? We are already dropping the display-specific attributes so I'm not sure it is worth keeping the meaning.

I totally agree it should be removed in the future, specially because there is also a description.

Reply:
Agreed offline